kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<archetype-descriptor name="${artifactId}" xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd"
+                      xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0"
+                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+  <requiredProperties>
+    <requiredProperty key="groupId">
+      <defaultValue>org.kie.businessapp</defaultValue>
+    </requiredProperty>
+    <requiredProperty key="artifactId">
+      <defaultValue>myserviceapp</defaultValue>
+    </requiredProperty>
+    <requiredProperty key="version">
+      <defaultValue>1.0</defaultValue>
+    </requiredProperty>
+    <requiredProperty key="archetypeCatalog">
+      <defaultValue>local</defaultValue>
+    </requiredProperty>
+    <requiredProperty key="appServerAddress">
+      <defaultValue>localhost</defaultValue>
+    </requiredProperty>
+    <requiredProperty key="appServerPort">
+      <defaultValue>8090</defaultValue>
+    </requiredProperty>
+    <!-- used to set up the controller in dev mode -->
+    <requiredProperty key="jbpmConsolePort">
+      <defaultValue>8080</defaultValue>
+    </requiredProperty>
+    <requiredProperty key="package">
+      <defaultValue>${groupId}</defaultValue>
+    </requiredProperty>
+    <requiredProperty key="kieVersion">
+      <defaultValue>${project.version}</defaultValue>
+    </requiredProperty>
+    <!-- appType options are bpm,brm,planner -->
+    <requiredProperty key="appType">
+      <defaultValue>bpm</defaultValue>
+    </requiredProperty>
+    <!-- dbType options are h2,mysql,postgres -->
+    <requiredProperty key="dbType">
```

Comment:
as mentioned, I'd removed it and generate maven profiles instead with duplicates of application.properties per data base - mysql and postgres